src/tallies/TallyBase.C

```diff
@@ -337,6 +342,10 @@ TallyBase::storeResults(const std::vector<unsigned int> & var_numbers,
                         unsigned int global_score,
                         const std::string & output_type)
 {
+  if (_use_displaced)
```

Comment:
This currently would require the user to have their input like

```
[Problem]
  type = OpenMCCellAverageProblem
  use_displaced_mesh = true

  [Tallies]
    [c]
      type = CellTally
      use_displaced_mesh = true
    []
  []
[]
```

duplicate input syntax can be error-prone and tedious. Recommend to add an accessor method to `OpenMCCellAverageProblem` to return to you `_use_displaced` and then use it instead of adding another input parameter.